ui/components/component-library/avatar-token/README.mdx

```diff
@@ -0,0 +1,32 @@
+import { Story, Canvas, ArgsTable } from '@storybook/addon-docs';
+
+import { AvatarToken } from './avatar-token';
+
+
+# AvatarToken
+
+The ` AvatarToken` is a component responsible for display of the image of a given token
+
+<Canvas>
+  <Story id="ui-components-component-library-base-avatar-base-avatar-stories-js--default-story" />
+</Canvas>
+
+## Props
+
+The `AvatarToken` accepts all props below as well as all [BaseAvatar](/ui-components-component-library-base-avatar-base-avatar-stories-js--default-story) component props.
+
+<ArgsTable of={AvatarToken} />
```

Comment:
Hey @georgewrmarshall, I am not sure why this is happening because I can't see any issues in the terminal wrt this. But at the same time, I am also having this problem in storybook.

Reply:
@georgewrmarshall I believe there is some conflict in the README configuration, if I don't import it, then I am able to see the props table.
<img width="1512" alt="Screenshot 2022-08-12 at 12 34 17 AM" src="https://user-images.githubusercontent.com/39872794/184219282-8c50fd6e-6c17-49b1-a7f9-33a15367eb06.png">

Reply:
Fixed! 🚀